src/main/drivers/pwm_output.c

```diff
@@ -459,6 +503,14 @@ void pwmWriteDshotCommand(uint8_t index, uint8_t motorCount, uint8_t command, bo
             pwmCompleteDshotMotorUpdate(0);
         }
         delayMicroseconds(delayAfterCommandUs);
+#ifdef USE_DSHOT_TELEMETRY
+        if (command == DSHOT_CMD_SIGNAL_LINE_CONTINUOUS_ERPM_TELEMETRY) {
+            dshotTelemetryEnabled = true;
```

Comment:
The issue is that the command needs to be sent repeatedly to become effective. This is true for many commands and has always been done in pwmWriteDshotCommand for the synchronous case. Therefore a state change that needs to happen after the sequence is complete fits better here.

Reply:
Other issue is that I needed it to work with dshot_prog from the cli. Would you really prefer that I set this flag which is a state internal to the dshot driver communication externally from each caller and intercept it at the generic cli dshot_prog command as well?

Reply:
In order to avoid the situation where both sides of the line are driven?

I think this case can best be dealt with by filtering out these commands in CLI - allowing CLI commands to drive unrelated parts of the configuration is not good practice, and will require us to run a `validateAndFixConfig()` for every `dshotprog` command to disable ERPM telemetry if burst Dshot is enabled.

Reply:
You know what? I think you might have misunderstood what this flag does. This doesn't switch erpm telemetry on and off. It just dynamically changes whether we listen for telemetry based on whether we have enabled in on the esc via the command. Does that make more sense? It's not a config change.

Reply:
To me, whether the flight controller does something (listening for telemetry), or does not do it is very much something that is part of the configuration, and should be properly controlled by a parameter.

Reply:
It's probably a moot point. I think I had to use this in the early versions when we still used a pull-up on the dshot line and blheli32 got confused by the resulting waveform when I switched to input and it wasn't in continuous telemetry mode. It might well be possible not to have this at all now. I will check it out.

Reply:
Yes, I could just eliminate this flag altogether and it works now.